nrfs/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_NRFS)
+    zephyr_include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
Things like that (Zephyr specific) should not be here but in the main Zephyr repository. See how this is done for nrfx in `modules/hal_nordic/nrfx/CMakeLists.txt`.